footer.php

```diff
@@ -64,16 +62,16 @@
 				</div><!-- span3 -->
 	
 	
-				<div class="span3">
+				<nav class="span3 bc-footer">
```

Comment:
If you are using more than one `<nav>` on a given page, **all** navs need to be labelled.

Reply:
I meant to delete that nav, thanks for the catch. Deleted in a commit.